test/documentation/seqan3_doxygen_cfg.in

```diff
@@ -51,3 +51,4 @@ FILE_PATTERNS          = *
 EXTRACT_PRIVATE        = ${SEQAN3_DOXYGEN_EXTRACT_PRIVATE}
 ENABLED_SECTIONS       = ${SEQAN3_DOXYGEN_ENABLED_SECTIONS}
 EXCLUDE_SYMBOLS        += ${SEQAN3_DOXYGEN_EXCLUDE_SYMBOLS}
+TOC_INCLUDE_HEADINGS   = 2
```

Comment:
Dont add this option under "Developer VS user mode" because it's not related to that. I have already added it further up, but commented out. Just remove the #